.bacon.yml

```diff
@@ -1,57 +1,57 @@
 test_suites:
-    - name: lint
-      script_path: /root/okta/okta-signin-widget/scripts
-      sort_order: '1'
-      timeout: '10'
-      script_name: lint
-      criteria: MERGE
-      queue_name: small
-    - name: unit-karma
-      script_path: /root/okta/okta-signin-widget/scripts
-      sort_order: '2'
-      timeout: '15'
-      script_name: unit
-      criteria: MERGE
-      queue_name: small
-    - name: unit-jest
-      script_path: /root/okta/okta-signin-widget/scripts
-      sort_order: '3'
-      timeout: '15'
-      script_name: jest
-      criteria: MERGE
-      queue_name: small
-    - name: testcafe
-      script_path: /root/okta/okta-signin-widget/scripts
-      sort_order: '4'
-      timeout: '20'
-      script_name: testcafe
-      criteria: MERGE
-      queue_name: small
-    - name: e2e
-      script_path: /root/okta/okta-signin-widget/scripts
-      sort_order: '5'
-      timeout: '10'
-      script_name: e2e
-      criteria: MERGE
-      queue_name: small
-    - name: e2e-saucelabs-windows
-      script_path: /root/okta/okta-signin-widget/scripts
-      sort_order: '6'
-      timeout: '20'
-      script_name: e2e-windows
-      criteria: MERGE
-      queue_name: small
-    - name: e2e-oie
-      script_path: /root/okta/okta-signin-widget/scripts
-      sort_order: '5'
-      timeout: '10'
-      script_name: e2e-oie
-      criteria: MERGE
-      queue_name: small
-    - name: publish
-      script_path: /root/okta/okta-signin-widget/scripts
-      sort_order: '7'
-      timeout: '10'
-      script_name: publish
-      criteria: MERGE
-      queue_name: small
\ No newline at end of file
+      - name: lint
+        script_path: /root/okta/okta-signin-widget/scripts
+        sort_order: "1"
+        timeout: "10"
+        script_name: lint
+        criteria: MERGE
+        queue_name: small
+      - name: unit-karma
```

Comment:
Is this an intended change?

Reply:
sort of.. I had config my IDE to auto format files (per file type) hence the changes..
revert it to minimize the changes.